### DISCUSSION TITLE
Custom query section

### DISCUSSION THREAD

**Discussion by KiddyTheKid** [KRTirtho/spotube#443] [Q&A] [answered]
Since the last update (v2.7) the custom query option is not visible. Is there a way to enable it back?

**Comment by KRTirtho** [comment, marked as answer]
Custom Query was removed in v2.7.0 in favor of Alternative Track Source and Spotube track-match public cache server

**Reply by KiddyTheKid** [reply]
Oh I get it. Thank you for your reply. I used to set "LYRICS" at the end of the query so i could get non interrupted music hahahaha. That's why I was interested in getting it back.

**Reply by KRTirtho** [reply]
We've sponsorblock support, which almost always cuts off all the non-music part of a track

**Reply by KiddyTheKid** [reply]
I will give that setting a try. Thank you for taking the time to reply. 
I forgot to mention that in am not being interrupted by sponsor content but for some parts that official video clips have. 

For example: 
Blinding Lights - The Weeknd

**Reply by KRTirtho** [reply]
SponsorBlock works for those too

**Reply by KiddyTheKid** [reply]
Yeees! It works great! Thank you.
